presidio-analyzer/presidio_analyzer/predefined_recognizers/__init__.py

```diff
@@ -28,4 +29,5 @@
            "UsItinRecognizer",
            "UsPassportRecognizer",
            "UsPhoneRecognizer",
-           "UsSsnRecognizer"]
+           "UsSsnRecognizer",
+           "TextAnalyticsRecognizer"]
```

Comment:
if the text analytics is a 'recognizer'
then you should not pass 'enable_text_analytics_recognizer=enable_text_analytics_recognizer' (L83 in main.py file)
instead you should pass it to the recognizer store...

if its not a recognizer, then it should be renamed and treated as such